src/content-handlers/iiif/modules/uv-contentleftpanel-module/ContentLeftPanel.ts

```diff
@@ -457,9 +471,16 @@ export class ContentLeftPanel extends LeftPanel<ContentLeftPanelConfig> {
     //   height = this.config.options.oneColThumbHeight;
     // }
 
-    const thumbs: Thumb[] = <Thumb[]>this.extension.helper.getThumbs(90);
+    let thumbs: Thumb[] = <Thumb[]>this.extension.helper.getThumbs(90);
     // this.extension.helper.getThumbs(width, height)
 
+    thumbs = thumbs.map(
+      (t) => {
+        t.uri += '?t=' + this.thumbsCacheKey;
```

Comment:
Maybe worth checking if `t.uri` already includes a `?` and use `&` in that case. Also perhaps better to use a more distinctive parameter like `_uv_t` to avoid clashing with some existing parameter.

Reply:
ok, as I thought. See my comment ;-)

Reply:
Yes, my comment was really a reply to your comment... just thought it would be helpful to attach it to the relevant line in the code. ;-)